### DISCUSSION TITLE
Is it possible to switch between server side full list and client side default filtering?

### DISCUSSION THREAD

**Discussion by Andrei-** [TarekRaafat/autoComplete.js#320] [Q&A]
I can show the full list returned from server by using this:
`searchEngine: (query, record) => { return record; }` (the answer provided here did not work for me: https://github.com/TarekRaafat/autoComplete.js/discussions/310)

But I need to be able to switch and let the default `searchEngine: 'strict'` do it's job when the user input is not digits.

For example if the user is searching by a number I need to show the full list, using:
`searchEngine: (query, record) => { return record; }`
But if the user is searching by letters, I just need to let the default engine logic take over
`searchEngine: 'strict'`

However, once I override the searchEngine setting, there is now way back.

**Comment by folknor** [comment]
You can just import the standard function, like for example

```js
import search from "@tarekraafat/autocomplete.js/src/controllers/searchController"
return search("Testing", "1234", {
	diacritics: false,
	highlight: false,
	mode: "strict"
})

```
